GNUmakefile

```diff
@@ -36,6 +38,10 @@ GOTESTPKGS ?= ./builder ./cgo ./compileopts ./compiler ./interp ./transform .
 # tinygo binary for tests
 TINYGO ?= $(call detect,tinygo,tinygo $(CURDIR)/build/tinygo)
 
+# isntalltion location defaults
+INSTALL_LOCAL_DIR ?= $(HOME)/go/bin/
```

Comment:
Have you considered providing the old-school standard variables and targets that packagers rely on and experienced "make" users expect?  

In particular, I think autoconf-generated configure scripts let users control installation location in two ways
```
make install prefix=/alternate/directory
make install DESTDIR=/alternate/directory
```
Both are needed because the first method works on windows (as it replaces the whole directory including drive letter), and the second is expected by e.g. dh_auto_install.

Debian packagers would thank you :-)

See https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.72/html_node/Installation-Names.html
and e.g. https://manpages.debian.org/testing/debhelper/dh_auto_install.1.en.html

Reply:
Gosh darn it, now I remember: GNUmakefile is only for the static build.  "Make install" should copy build/TinyGo, not do "go install". 

Also, using profile and DESTDIR must stack; DESTDIR applies after profile, and is for package wranglers only.

Reply:
Thanks @dkegel-fastly for the feedback. I should really adapt this patch to the convention utilizing `prefix` and `DESTDIR`.

Reply:
also fix the typos in the comments else the dreaded Spelling Police will come after you :-)